src/tests/logic/calculate.test.js

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+import calculate from '../../logic/calculate';
```

Comment:
- [ ] Based on the requirements of this project, all the functionalities, and scenarios of the calculate.js module should be covered. We shall kindly consider the different scenarios below:

-  if the button name(buttonName) === 'AC'
- ------------------------------------------  === '.'
- ------------------------------------------  === '='
- ------------------------------------------  === '+/-'
- ------------------------------------------  === '0'